sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/CategorizedEntity.java

```diff
@@ -10,43 +10,21 @@
  */
 @Immutable
 public final class CategorizedEntity {
-    /*
-     * CategorizedEntity text as appears in the request.
-     */
     private final String text;
-
-    /*
-     * CategorizedEntity category, such as Person/Location/Org/SSN etc.
-     */
     private final EntityCategory category;
-
-    /*
-     * CategorizedEntity sub category, such as Age/Year/TimeRange etc.
-     */
     private final String subcategory;
-
-    /*
-     * Start position for the entity text.
-     */
+    private final double confidenceScore;
```

Comment:
nit: was just wondering why the comments disappeared from here

Reply:
Trying to make the model consistency that don't have comments on the private properties.